### DISCUSSION TITLE
Azure container instance instructions are missing a critical port

### DISCUSSION THREAD

**Discussion by rodneytamblyn** [datalust/seq-tickets#2151] [General]
When setting up a container instance open ports should include port 5341.  I wasted quite a bit of time today trying to figure this one out.

For any others that encounter this, If you are seeing errors like this in your NodeJS app when attempting to setup Winston logging:

```
'TypeError: fetch failed\n' +
    '    at Object.processResponse (node:internal/deps/undici/undici:6406:34)\n' +
    '    at node:internal/deps/undici/undici:6727:42\n' +
    '    at node:internal/process/task_queues:140:7\n' +
    '    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)\n' +
    '    at AsyncResource.runMicrotask (node:internal/process/task_queues:137:8)\n' +
    '    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {\n' +
```
... it may be because the port is not open or the server is unreachable.  I went down the rabbit hole of thinking this was an issue with undici (as there are a number of threads on fetch failing this way) but in this case the cause was simpler.

On https://docs.datalust.co/docs/using-azure-container-instances-aci please update container create step to include port 5341
```
az container create \
....
  --ports 80 443 5341 \
  ```

**Comment by rodneytamblyn** [comment]
Further to https://docs.datalust.co/docs/https-tlsssl
"Seq always uses 443 as the HTTPS API port and 45341 as the HTTPS ingestion port."
... but I think these will only work if HTTPS is configured (which the container instance setup instructions don't include advice to complete.  Can you confirm?  I haven't yet configured https and 443 port doesn't work for me for ingestion.

If 443 is the correct port to use for API ingestion only if HTTPS is configured then perhaps make this clearer.

**Reply by liammclennan** [reply]
Hi @rodneytamblyn ,

See https://docs.datalust.co/docs/docker-deployment-overview

You do not need to expose port 5341 because port 80 is the API and ingestion port. However, you will need to expose 5341 if you are trying to connect on 5341. 

You are correct that the HTTPS ports will not work unless HTTPS is configured.

**Comment by rodneytamblyn** [comment]
Message ID: ***@***.***>https://github.com/datalust/winston-seq

Example config shows port 5341, that’s what I was going from.

    new SeqTransport({
      serverUrl: "https://your-seq-server:5341",
